### PR TITLE
LPD-25553 add disabled antisamy tests for LPD-24898

### PR DESCRIPTION
**Pull request by cdbm**

### CODE REVIEW

**Comment by cdbm** [comment]
ci:test:sf

**Comment by cdbm** [comment]
ci:test:relevant

**Comment by cdbm** [comment]
ci:test:stable

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/antisamy/AntiSamy.testcase [hunk 1]
```diff
@@ -176,4 +217,78 @@ definition {
 		}
 	}
 
+	@description = "LPD-25553: Assert AntiSamy is correctly disabled"
+	@priority = 3
+	test ScriptInWikiPortletShouldNotBeSanitizedWhenAntiSamyIsDisabled {
+		property antisamy.enabled = "true";
```
Comment (ChrisKian) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/antisamy/AntiSamy.testcase:223]: Could we just change this value to `false` and remove the "Disable AntiSamy" task?

**Comment by ChrisKian** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/antisamy/AntiSamy.testcase [hunk 2]
```diff
@@ -61,6 +61,47 @@ definition {
 		}
 	}
 
+	@description = "LPD-25553: AntiSamy Should be disabled without exceptions"
+	@priority = 3
+	test AntiSamyShouldBeDisabledWithoutExceptions {
+		property antisamy.enabled = "true";
```
Comment (ChrisKian) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/antisamy/AntiSamy.testcase:67]: Looks likt this property is used to initialize a config file.  From build.xml:
		<get-testcase-property property.name="antisamy.enabled" />

```
		<if>
			<equals arg1="${antisamy.enabled}" arg2="true" />
			<then>
				<antcall target="prepare-antisamy-config-file" />

				<gradle-execute dir="${project.dir}/modules/apps/portal-security/portal-security-antisamy" task="deploy">
					<arg value="clean" />
				</gradle-execute>
			</then>
		</if>
```

**Comment by ChrisKian** [comment]
Forwarded here: https://github.com/liferay-appsec/liferay-portal/pull/1760

**Pull request closed without merging**